src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -343,7 +343,10 @@ public synchronized void applyDDL(Sequenced<DDLEvent> sequencedEvent) throws Exc
   private void handleDDL(DDLEvent event, String normalizedDatabaseName, String normalizedTableName,
                          String normalizedStagingTableName)
     throws IOException, DeltaFailureException, InterruptedException {
-
+    LOG.info("DDL Event={}, sequenceNumber={}", event, latestSequenceNum);
+    if (event.getSchema() != null) {
```

Comment:
Debug logging should be guarded by LOG.isDebugEnabled()

```suggestion
    if (LOG.isDebugEnabled() && event.getSchema() != null) {
```